src/utils/ImportExportHelper.ts

```diff
@@ -0,0 +1,59 @@
+// Utility methods to call import/export Jahia API
```

Comment:
Any particular reason for calling this file ImportExportHelper considering there's currently only an export method? 

i.e. what if, at the time an import method is implemented later in the future, we figure out this import method should have its own file, then we might end up having both importExportHelper without an import method, and an importHelper.

Reply:
Initially it was supposed to handle both, but the import side doesn't exists. 
File renamed